src/SFA.DAS.Notifications.CloudService/ServiceDefinition.csdef

```diff
@@ -35,6 +36,10 @@
       <Setting name="EmailServiceName" />
       <Setting name="LoggingRedisConnectionString"/>
       <Setting name="LoggingRedisKey"/>
+      <Setting name="InstrumentationKey" />
     </ConfigurationSettings>
+    <Endpoints>
```

Comment:
That is a workaround for an issue we have with deployment of cloud services with worker roles